.ci/pipeline_definitions

```diff
@@ -3,23 +3,24 @@
 # SPDX-License-Identifier: Apache-2.0
 
 gardener-extension-shoot-rsyslog-relp:
-  template: 'default'
   base_definition:
-    repo: ~
     traits:
       version:
         preprocess: 'inject-commit-hash'
+      component_descriptor:
+        ocm_repository: europe-docker.pkg.dev/gardener-project/snapshots
```

Comment:
Same as https://github.com/gardener/gardener-extension-registry-cache/pull/113#discussion_r1426939667

Reply:
@plkokanov : similar to gerc: if snapshot-versions from `master` should still be possible to be deployed to public landscapes, OCM Component Descriptors are required. It would however make sense, probably, to consequently mv generation/publishing of OCM Component Descriptors also into prow-pipeline (I think we should discuss this separately from this specific pull-request, considering this pull-request does not actually change anything here (if no ocm-repository is configured, a default (residing on GCR) is used).